.github/workflows/build.yml

```diff
@@ -7,48 +7,42 @@ jobs:
   build:
     runs-on: ubuntu-latest
     # Consistent with base image in Dockerfile
-    container: stepik/hyperstyle-base:py3.8.11-java11.0.11-node14.17.3
+    container: nastyabirillo/hyperstyle:1.2.1
 
     steps:
       - name: Install git
         run: |
           apt-get update
           apt-get -y install git
-    
-      - name: Checkout
-        uses: actions/checkout@v2
 
-      - name: Setup venv
+      - name: Check env variables
         run: |
-          python -m venv ./main/venv
+          echo $DETEKT_DIRECTORY && echo $DETEKT_VERSION
```

Comment:
We need to update main readme too:
1. Remove hyperstyle manual installation
2. Add instruction how to add env variables
3. Add instruction how to install checkers 
Or maybe just redirect users to https://github.com/hyperskill/hyperstyle/blob/bf3c6e2dc42290ad27f2d30ce42d84a53241544b/Dockerfile